### DISCUSSION TITLE
model not working on ios 15 - displays update to latest ios message

### DISCUSSION THREAD

**Discussion by briscoe-31** [google/model-viewer#2826] [Q&A]
When viewing on iphone with ios 15 the models wont display. We just get a message saying update to the latest ios. But when checking for ios updates its already the laters version. 

We have tried the model on android and also iphones with ios14 on and they work fine. Has anyone had the same issue or know how to fix it?

**Comment by elalish** [comment]
Would you mind filing an issue for this, and filling out the whole template? There are definitely some problems with the new Safari 15, but we need as many  detailed repros as we can get in order to get to the bottom of it.

**Reply by rahulconsagous** [reply]
Please find below Video from both iOS 15 and iOS 14 Version.

iOS 15 Device Video [ Not working on that platform ]
https://drive.google.com/file/d/1xkiRi6L01yMXU4laGe4amH2GSS9t_Nn8/view?usp=sharing

iOS 14 [ Working ]
https://drive.google.com/file/d/1KdOGGNDo375G32ZxiQOIZatQc_DuD3v_/view?usp=sharing

**Comment by elalish** [comment]
I assume you're supplying your own USDZ with `ios-src`, since we don't support auto-generation for animations yet. In that case I'd guess some kind of backwards-incompatible change on Apple's side (@grorg do you know?). Perhaps try re-exporting your USDZ with Apple's latest OS?
